pom.xml

```diff
@@ -118,7 +118,7 @@
     <connection>scm:git:git://github.com/jenkinsci/${project.artifactId}-plugin.git</connection>
     <developerConnection>scm:git:git@github.com:jenkinsci/${project.artifactId}-plugin.git</developerConnection>
     <url>https://github.com/jenkinsci/${project.artifactId}-plugin</url>
-    <tag>HEAD</tag>
+    <tag>warnings-4.61</tag>
```

Comment:
Ah sorry, I might have changed the tag to be able to create an installable hpi for our server. Should I leave it as HEAD?
I will also check the test suite again.